doc/classes/CanvasItem.xml

```diff
@@ -569,13 +569,14 @@
 			The rendering layers in which this [CanvasItem] responds to [Light2D] nodes.
 		</member>
 		<member name="material" type="Material" setter="set_material" getter="get_material">
-			The material applied to textures on this [CanvasItem].
+			The material applied to this [CanvasItem].
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate" default="Color(1, 1, 1, 1)">
-			The color applied to textures on this [CanvasItem].
+			The color applied to this [CanvasItem].
 		</member>
 		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate" default="Color(1, 1, 1, 1)">
-			The color applied to textures on this [CanvasItem]. This is not inherited by children [CanvasItem]s.
+			The color applied to this [CanvasItem]. Does not propagate to children [CanvasItem]s.
```

Comment:
And so this can be adjusted to be more similar.
```suggestion
			The color applied to this [CanvasItem]. This property does [b]not[/b] affect child [CanvasItem]s, unlike [member modulate] which affects both the node itself and its children.
```